assets/javascripts/discourse-assign/initializers/extend-for-assigns.js

```diff
@@ -629,7 +639,7 @@ function initialize(api) {
               : "";
           return `<${tagName} class="assigned-to discourse-tag simple" ${href}>
         ${icon}
-        <span>${name}</span>
+        <span title="${note}">${name}</span>
```

Comment:
I didn't confirm this but I think we have to escape `note` here because it is user input.

Reply:
Good point thanks!